src/library/constants/constants.js

```diff
@@ -5,5 +5,15 @@ export const PROJECT_CODES = {
   status: { open: 90, test: 80 },
   policy: { private: 10, publicSummary: 50 },
 }
-
+export const apiDataTypes = {
+  benthicAttributes: 'benthic_attributes',
+  choices: 'choices',
+  collectRecords: 'collect_records',
+  fishFamilies: 'fish_families',
+  fishGenera: 'fish_genera',
+  fishSpecies: 'fish_species',
+  projectManagements: 'project_managements',
+  projectProfiles: 'project_profiles',
+  projectSites: 'project_sites',
+}
```

Comment:
this probably should have been an enum all along. I didnt yak shave here to refactor everything to use this, but its here for moving forward and opportunistic refactor. 

Also it looks like I forgot to freeze it. Will do.